x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -80,13 +80,67 @@ describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
       .should('have.text', createdHost.hostname);
   });
 
+  it('should update endpoint policy on Endpoint', () => {
+    cy.intercept({ pathname: HOST_METADATA_LIST_ROUTE }).as('metadataRequest');
+    loadPage(APP_ENDPOINTS_PATH);
+
+    const agentIdPath = 'response.body.data[0].metadata.agent.id';
+    const policyVersionPath =
+      'response.body.data[0].metadata.Endpoint.policy.applied.endpoint_policy_version';
+    cy.wait('@metadataRequest', { timeout: 30000 }).its(agentIdPath).should('be.a', 'string');
+    cy.get('@metadataRequest').its(policyVersionPath).should('be.a', 'string');
+    cy.get('@metadataRequest')
+      .its(agentIdPath)
+      .then((endpointId: string) => cy.get(`[data-endpoint-id=${endpointId}]`))
+      .as('endpointRow');
+    cy.get('@metadataRequest')
+      .its(policyVersionPath)
+      .should('be.a', 'string')
+      .then((policyVersion: string) => parseInt(policyVersion, 10))
+      .as('originalPolicyRevision');
+
+    cy.get('@endpointRow').findByTestSubj('policyNameCellLink').click();
+    cy.location('pathname', { timeout: 20000 }).should(
+      'match',
+      /^\/app\/security\/administration\/policy\/[a-f0-9-]+\/settings$/
+    );
+
+    // make a change to enable save
+    cy.getByTestSubj('endpointPolicyForm-malware-blocklist-enableDisableSwitch')
+      .should('be.visible')
+      .as('blocklistSwitch');
+    cy.get('@blocklistSwitch').click();
+    cy.get('@blocklistSwitch').should('be.enabled');
+
+    cy.getByTestSubj('policyDetailsSaveButton').should('be.visible').click();
+    cy.getByTestSubj('policyDetailsConfirmModal').should('be.visible').click();
+    cy.getByTestSubj('confirmModalConfirmButton').should('be.visible').click();
+    cy.contains(/has been updated/);
+    cy.getByTestSubj('policyDetailsBackLink').should('be.visible').click();
+    cy.location('pathname', { timeout: 20000 }).should('equal', APP_ENDPOINTS_PATH);
+
+    // Assert disappearing 'Out-of-date' indicator, Success Policy Status and increased revision number
+    cy.get('@endpointRow')
+      .findByTestSubj('rowPolicyOutOfDate', { timeout: 10000 })
+      .should('not.exist'); // depends on the 10s auto-refresh
+
+    cy.get('@endpointRow').findByTestSubj('policyStatusCellLink').should('contain', 'Success');
+
+    cy.get<number>('@originalPolicyRevision').then((originalRevision: number) => {
+      cy.get('@endpointRow')
+        .findByTestSubj('policyListRevNo')
+        .contains(originalRevision + 1);
```

Comment:
Not sure whether it would yield true for both `2` and `2x`, probably?

Reply:
Ah, good catch. Will fix.